content/sigs/cloud-native/pluggable-storage.adoc

```diff
@@ -29,73 +29,107 @@ link:/sigs/cloud-native[Cloud Native SIG Page].
 
 Below you can find a summary of ongoing activities and their current status:
 
+[frame="topbot",grid="all",options="header", cols="20%,50%,30%"]
 |=========================================================
-|Type | Status | JEPs | Comment
+|Type / Status | Comment | Implementation(s)
 
-| Artifacts
-| Available
-| jep:202[]
-|
+| **Artifacts** +
+  **(Available)**
+| Fully delivered, with support for uploading artifacts directly from agents.
+  Related JEPs: jep:202[].
 
-| Credentials
-| Available
-| N/A
-| Completed before the JEP process was introduced
+  link:/doc/developer/extensions/jenkins-core/#artifactmanagerfactory[Extension point]
 
-| Build logs
-| In progress
-| jep:207[], jep:210[], jep:212[]
-|
+| plugin:artifact-manager-s3[Artifact Manager on S3],
+  plugin:azure-artifact-manager[Azure Artifact Manager],
+  link:/doc/developer/extensions/jenkins-core/#artifactmanagerfactory[<More implementations>]
 
-| System logs
-| Available
-| N/A
+| **Credentials** +
+  **(Available)**
+| Completed before the JEP process was introduced.
+
+link:https://www.jenkins.io/doc/developer/extensions/credentials/#credentialsprovider[Extension point]
+| plugin:kubernetes-credentials-provider[Kubernetes Credentials Provider],
+  plugin:aws-secrets-manager-credentials-provider[AWS Secrets Manager Credentials Provider],
+  https://www.jenkins.io/doc/developer/extensions/credentials/#credentialsprovider[<More implementations>]
+
+| Build logs +
+  (Preview / Paused)
+| The story is completed for Pipeline Log Storage, only Jenkins Pipeline job types are supported.
```

Comment:
Not sure I would say JEP-210 is “completed”. In particular

* the change to make `sh` steps stream properly was reverted due to some minor issues and was never reapplied (it is forcibly enabled by `pipeline-cloudwatch-logs`)
* the log cursor in core (also used by Blue Ocean) is still a `long` rather than an opaque implementation-defined type, making it impossible to efficiently support streaming display from cloud services

Reply:
Right. "preview" then?

Reply:
```suggestion
| Pipeline Log Storage API and reference implementations are available for preview, only Jenkins Pipeline job types are supported.
```